examples/dreamcast/2ndmix/2ndmix.c

```diff
@@ -398,6 +398,11 @@ int load_pcx(char *pcxdata) {
     int num_bytes;
     struct  pcx_hdr pcxh;
 
+    if(image == NULL) {
+        printf("Image not allocated\r\n");
```

Comment:
```suggestion
        fprintf(stderr, "Image not allocated\r\n");
```
So I am always super anal about making sure that *error* messages go to `stderr` rather than `stdout`. It's not buffered, so it displays immediately, plus when we get testing automation set up on GitHub, build runners listen for messages on `stderr` when trying to diagnose issues with a build that can flag it as a failure.

Reply:
I'd just say that it's out of scope here. There's already plenty of error messages in 2ndmix and they all just go to printf. If it is to be changed it should be an overhaul for the whole file.

Where does dbglog fit into that scheme? We have quite a few examples using that too.